R/bfpackcommon.R

```diff
@@ -904,39 +930,46 @@
   if (is.null(bfs)) return()
 
   if (type %in% c("variances", "tTestMultiSamples")) {
-    title1 <- gettext("BF(0c)")
-    title2 <- gettext("BF(c0)")
+    title1 <- gettext("H0 vs. H±)")
+    title2 <- gettext("H± vs. H0")
     stdBfTable$addColumnInfo(name = "bf1", title = title1, type = "number")
     stdBfTable$addColumnInfo(name = "bf2", title = title2, type = "number")
     stdBfTable$setData(data.frame(bf1 = bfs[1], bf2 = 1/bfs[1]))
 
   } else {
 
     stdBfTable$addColumnInfo(name = "coefficient", title = "", type = "string")
-    stdBfTable$addColumnInfo(name = "bf0", title = gettext("BF(0u)"), type = "number")
-    stdBfTable$addColumnInfo(name = "bf1", title = gettext("BF(-u)"), type = "number")
-    stdBfTable$addColumnInfo(name = "bf2", title = gettext("BF(+u)"), type = "number")
-    stdBfTable$addColumnInfo(name = "bf3", title = gettext("BF(u0)"), type = "number")
-    stdBfTable$addColumnInfo(name = "bf4", title = gettext("BF(u-)"), type = "number")
-    stdBfTable$addColumnInfo(name = "bf5", title = gettext("BF(u+)"), type = "number")
+    stdBfTable$addColumnInfo(name = "bf0", title = gettext("Best vs. H0"), type = "number")
+    stdBfTable$addColumnInfo(name = "bf1", title = gettext("Best vs. H-"), type = "number")
+    stdBfTable$addColumnInfo(name = "bf2", title = gettext("Best vs. H+"), type = "number")
 
     if (type == "tTestIndependentSamples") {
       dtFill <- data.frame(coefficient = gettext("difference"))
+      bfs <- as.matrix(bfs)
     } else {
       dtFill <- data.frame(coefficient = rownames(bfs))
     }
-    dtFill[, c("bf0", "bf1", "bf2")] <- bfs
-    dtFill[, c("bf3", "bf4", "bf5")] <- 1/bfs
-    stdBfTable$setData(dtFill)
 
+    # we do the best performing hypothesis vs. the others
+    bfs <- t(apply(bfs, 1, function(x) max(x)/x))
+    out <- as.data.frame(bfs)
+    outDf <- sapply(out, function(x) sapply(x, function(y) {
+      if (y == 1) {
+        return(NA)
+      } else {
+        return(y)
+      }
+    }))
```

Comment:
```suggestion
    outDf <- sapply(out, function(x) {
    	x[x == 1] <- NA
    	x
    })
```
should be equivalent and faster? I'm assuming here that `x` is a vector of numbers and not a list where each element is a number.